pkg/sql/table_test.go

```diff
@@ -309,3 +314,33 @@ func TestPrimaryKeyUnspecified(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 }
+
+func TestCanCloneTableWithUDT(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+
+	ctx := context.Background()
+	params, _ := tests.CreateTestServerParams()
+	s, sqlDB, kvDB := serverutils.StartServer(t, params)
+	defer s.Stopper().Stop(ctx)
+	if _, err := sqlDB.Exec(`
+SET experimental_enable_enums=true;
+CREATE DATABASE test;
+CREATE TYPE test.t AS ENUM ('hello');
+CREATE TABLE test.tt (x test.t);
+`); err != nil {
+		t.Fatal(err)
```

Comment:
why not use testify :P ?

Reply:
What would that help with here?

Reply:
tests are easier to read and write. no biggie though.

Reply:
Sorry, I meant in this particular case -- (i dont use testify often)